i18n/en.json

```diff
@@ -2275,7 +2275,7 @@
   "integrations.help.appDirectory": "App Directory",
   "integrations.incomingWebhook.description": "Incoming webhooks allow external integrations to send messages",
   "integrations.incomingWebhook.title": "Incoming Webhook",
-  "integrations.oauthApps.description": "OAuth 2.0 allows external applications to make authorized requests to the Mattermost API.",
+  "integrations.oauthApps.description": "OAuth 2.0 allows external applications to make authorized requests to the Mattermost API",
```

Comment:
nit: this change should be reflected too in the `defaultMessage` of the file [components/integrations/integrations.jsx (line 148)](https://github.com/mattermost/mattermost-webapp/blob/master/components/integrations/integrations.jsx#L148), where we make use of this key

Reply:
Done.